policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/stream/ConsumerTopicConfiguration.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.policy.stream;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import uk.gov.gchq.palisade.Generated;
+import uk.gov.gchq.palisade.service.policy.stream.ProducerTopicConfiguration.Topic;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Configuration for Akka Kafka consumer
+ */
+@ConfigurationProperties("akka.kafka.consumer")
+public class ConsumerTopicConfiguration {
+    private String discoveryMethod;
+    private String serviceName;
+    private String useDispatcher;
+    private Map<String, String> kafkaClients = new HashMap<>();
+    private Map<String, String> connectionChecker = new HashMap<>();
+    private Map<String, ProducerTopicConfiguration.Topic> topics = new HashMap<>();
```

Comment:
We've imported Topic , use that rather than the longer ProducerTopicConfig.Topic used here

Reply:
changed